packages/graphql/lib/get-graphql-error.js

```diff
@@ -0,0 +1,36 @@
+const getHeaders = (status, headers) => {
+  if (status === 429) {
+    const retryAfter = headers.get('retry-after');
+    return retryAfter ? { 'retry-after': retryAfter } : {};
+  }
+  return {};
+};
+
+module.exports = error => {
+  if (!error.networkError || !error.networkError.response) {
+    return error;
+  }
+
+  const { stack, response } = error.networkError;
+  const { status: statusCode, statusText, headers: responseHeaders } = response;
+  const status = statusCode === 429 ? 429 : 500;
```

Comment:
I think we need the same logic for redux so in my opinion we should just throw a new error with all useful information that we have here.

Reply:
Sorry, I don't understand that. Isn't this what is happening here?

Reply:
We have opinionated logic here about the status code and error message. I think it would be beneficial to move that logic somewhere else in order to have the same feature for redux actions.

As we can't override this anymore I would either create a mixin hook or just don't handle the status codes here at all. Responding with a status code is responsibility of an express middleware IMO - that we want take the status code of this request into account is an implementation detail of that middleware.